client/src/components/FriendsRecommendations/FriendsRecommendations.stories.tsx

```diff
@@ -6,30 +6,35 @@ import {FriendsRecomendations} from './index';
 
 export const FriendsRecomendationsState = {
     loggedUsername: 'Maks',
+    loadFollow: false,
```

Comment:
what the responsibility of this var?

Reply:
No responsibility . Sorry i'm inattentive.